lib/block/block.js

```diff
@@ -277,7 +277,7 @@ Block.prototype._getHash = function() {
 
 var idProperty = {
   configurable: false,
-  writeable: false,
+  enumerable: true,
```

Comment:
This is used for both `id`, and `hash` we may not want these both to be enumerable.